Resources/Prototypes/_EE/Entities/Structures/Piping/Atmospherics/binary.yml

```
@@ -0,0 +1,34 @@
+﻿- type: entity
+  parent: GasPressurePump
+  id: GasPressurePumpHighFlow
+  name: high flow gas pump
+  description: A pump that moves gas by pressure. It pumps air at three times the pressure of a standard pump.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: GasPressurePump
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)